en/docs/consume/manage-application/generate-keys/obtain-access-token/access-tokens-per-device.md

```diff
@@ -1,8 +1,11 @@
 # Access Tokens Per Device
 
-WSO2 API Manager returns the same token repeatedly if a valid token exists for the requesting Application, on behalf of the user. However, the latter mentioned scenario becomes an issue if the same user is using the same Application in two devices (e.g., If you have two instances of the same Application running on your iPhone and iPad, and your iPhone already has a token on behalf of you, your iPad will get the same token if you requested for it within the same validity period. Therefore, if one of your devices revoke this token (e.g., revoke on logout), the token that you obtained for your other device becomes invalid as the devices use the identical tokens.
+In WSO2 API Manager, the behavior for generating JWT tokens with the same application keys and scopes is different to the previous approach with Opaque tokens. For JWT tokens, WSO2 API Manager no longer returns the same existing valid JWT token for each token request if the associated application, user and requested scopes are the same. Additionally, if a valid token already exists for the requesting application, user and requested scopes, generating a new token will result in the revocation of the previous token associated with that application and user.
+
+However, this change in behavior is particularly relevant when considering scenarios where the same user is utilizing the same application across multiple devices. For example, if a user has the same application installed on both their iPhone and iPad, and a token has been generated on the iPhone, requesting a token on the iPad within the same validity period could result in obtaining the same token. Consequently, if one of these devices revokes the token, such as during a logout operation, the token obtained for the other device will also become invalid. This is because both devices are utilizing identical tokens.
```

Comment:
@ruthryi this is wrong. Both wont be using identical tokens.

The idea  is, if token is generated by iPhone and if iPad is generating the token for same application, user and scopes, then iPhone's token will be revoked. iPad will also get a new token.

Reply:
Done